static/resources.json

```diff
@@ -155,6 +139,144 @@
       }
     }
   },
+  "Books": {
+    "description": "The best books for learning Python or Python Frameworks",
+    "resources": {
+      "Automate the Boring Stuff with Python": {
```

Comment:
There's also a Udemy course for this. Can we add that too? 

Here's a link with a coupon added that is always going to cost 10 dollars.

https://www.udemy.com/automate/?couponCode=FOR_LIKE_10_BUCKS